### DISCUSSION TITLE
Fingerprint login not working

### DISCUSSION THREAD

**Discussion by Cod3Bende4** [pop-os/pop#3141] [Q&A]
Hi, I am having following device for fingerprint login in my PC but fingerprint option is not showing in the users tab. Can anyone help me on this ??

`Bus 003 Device 003: ID 27c6:589a Shenzhen Goodix Technology Co.,Ltd. FingerPrint`

**Comment by jacobgkau** [comment]
What laptop brand and model is this?

I found a reference to Dell publishing Linux drivers for some Goodix fingerprint readers:
- https://askubuntu.com/questions/1341912/goodix-fingerprint-driver-linux 
- https://www.dell.com/community/en/conversations/linux-general/xps-13-9300-does-fingerprint-reader-work-on-linux/647f86e5f4ccf8a8de60605e?commentId=647f8989f4ccf8a8de9314cd#M63982
